### DISCUSSION TITLE
Fade in image when it loads with <b-img-lazy>

### DISCUSSION THREAD

**Discussion by Flawe** [bootstrap-vue/bootstrap-vue#6210] [Q&A] [answered]
Is it possible to fade between blank-src and src when using <b-img-lazy> and the src image loads? By default the image pops in, and this can be a bit jarring, especially if there's multiple instances on screen.

**Comment by Hiws** [comment, marked as answer]
I don't think there's a built-in way to accomplish this.

However you can create your own little custom component to handle this, by using the `v-b-visible` directive and the `<transition>` element. This way you can swap between two `<b-img>` which will trigger a transition.

Here's an example:
https://codepen.io/Hiws/pen/XWjRJoQ

**Reply by Flawe** [reply]
Beautiful, thank you @Hiws
